js/sykmeldinger/data/dine-sykmeldinger/dineSykmeldingerSagas.js

```diff
@@ -11,17 +11,56 @@ import {
     SYKMELDING_GJENAAPNET,
     SYKMELDING_SENDT,
 } from '../din-sykmelding/dinSykmeldingActions';
+import { toggleSykmeldingerBackend, unleashtogglesHentetSelector } from '../../../data/unleash-toggles/unleashTogglesSelectors';
+import { HENTET_UNLEASH_TOGGLES } from '../../../data/unleash-toggles/unleashToggles_actions';
+import { erNaisLabsDemo } from '../../../utils/urlUtils';
 
 const { HENT_DINE_SYKMELDINGER_FORESPURT } = actions;
 
+// Because envvars can not easily be injected in build
+const getSykmeldingerBackendUrl = () => {
+    const url = window
+    && window.location
+    && window.location.href
+        ? window.location.href
+        : '';
+
+    if (url.indexOf('tjenester.nav') > -1) {
+        // Prod
+        return 'https://sykmeldinger-backend-proxy.dev.nav.no/api/v1/syforest';
```

Comment:
dette er ikke prod :)

Reply:
Dere må også sette opp DNS.  https://sykmeldinger-backend-proxy.nav.no/isAlive svarer ikke

Anbefaler også nyere versjon av flex-proxy, varianten dere kjører er litt alpha og slet med connections, satt opp for mange på nytt som var uheldig over lengre avstand, bytta over til en spring variant (som trenger mer minne). Se repoet, skal ish holde å bumpe cpu, minne og FROM i Dockerfile

Reply:
Og endre fra epxress.js til spring syntax i routes.yaml